database/migrations/2021_05_18_020534_add_min_value_to_depreciation.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddMinValueToDepreciation extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('depreciations', function (Blueprint $table) {
+            $table->string('min_value')->after('months')->nullable();
```

Comment:
I think we had discussed changing this to decimal, no?

Reply:
(We should also add `->default(null)` in there too, as different versions of different databases handle nullables slightly differently.)

Reply:
We should also add `->default(null)` in there, as different versions of different databases handle nullable fields slightly differently, so it's better to be explicit.